cli/src/alluxio.org/cli/cmd/conf/get.go

```diff
@@ -0,0 +1,68 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package conf
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/palantir/stacktrace"
+	"github.com/spf13/cobra"
+
+	"alluxio.org/cli/env"
+	"alluxio.org/log"
+)
+
+var Get = &GetCommand{
+	BaseCommand: &env.BaseCommand{
+		CommandName:   "get",
+		JavaClassName: "alluxio.cli.GetConf",
+		ShellJavaOpts: fmt.Sprintf(env.JavaOptFormat, env.ConfAlluxioConfValidationEnabled, false),
+	},
+}
+
+type GetCommand struct {
+	*env.BaseCommand
+}
+
+func (c *GetCommand) Base() *env.BaseCommand {
+	return c.BaseCommand
+}
+
+func (c *GetCommand) ToCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("%v [key]", Get.CommandName),
```

Comment:
I'm surprise we have to write out the the "help" format ourselves. In python for example, when creation an argument, one can specify if it is required or optional, repeated, etc... when creating the option. The python library then take care of generation the help menu. 
Is there a way to do that here? I worry this will lead to a great fragmentation and inconsistency of help menus, which is one of the main goals of rewriting the cli in Go (i.e. consistency and repeatability).

Reply:
help info for flags are autogenerated, but not for args. this is what the help menu looks like without specifying the `[key]` portion of the Use string
```
$ bin/cli.sh conf get -h
Look up a configuration value by its key or print all configuration if no key is provided

Usage:
   conf get [flags]

Flags:
      --attach-debug        True to attach debug opts
  -h, --help                help for get
  -D, --java-opts strings   Alluxio properties to apply, ex. -Dkey=value

Global Flags:
      --debugLog   True to enable debug logging
```
so it makes sense that we have to describe the arguments. in this example, i've specified `java-opts` as a repeatable flag which cobra indicates as `strings` rather than `string`

Reply:
help info for flags are autogenerated, but not for args. this is what the help menu looks like without specifying the `[key]` portion of the Use string
```
$ bin/cli.sh conf get -h
Look up a configuration value by its key or print all configuration if no key is provided

Usage:
   conf get [flags]

Flags:
      --attach-debug        True to attach debug opts
  -h, --help                help for get
  -D, --java-opts strings   Alluxio properties to apply, ex. -Dkey=value

Global Flags:
      --debugLog   True to enable debug logging
```
so it makes sense that we have to describe the arguments. in this example, i've specified `java-opts` as a repeatable flag which cobra indicates as `strings` rather than `string` for a simple string flag